keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -379,14 +385,37 @@ command or function to work in the same way in another tool. This can
 further enhance the usability and adoption of different cluster
 manager.
 
-### Notes/Constraints/Caveats (Optional)
+### Notes/Constraints/Caveats
+#### What's the relationship between the clusterProfile API and cluster Inventory?
+The clusterProfile API represents a single member cluster in a cluster Inventory.
+
+#### What's the relationship between a cluster Inventory and clusterSet?
+A cluster inventory is considered a clusterSet if all its member clusters adhere to the 
+[namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md) principle.
+
+#### How should the API be consumed?
+We recommend that all clusterProfile objects within the same cluster inventory reside on 
+a dedicated hub Kubernetes cluster. This approach allows consumers to have a single integration 
+point to access all the information within a cluster inventory. Additionally, a multi-cluster aware
+controller can be run on the hub cluster to offer high-level functionalities over this inventory of clusters.
+
+####  How should we organize clusterProfile objects on a hub cluster?
+While there are no strict requirements, we suggest making the clusterProfile API a namespace-scoped object. 
+This approach allows users to leverage Kubernetes' native namespace-based RBAC if they wish to restrict access to 
+certain clusters within the inventory.  
+
+However, if the cluster inventory is a clusterSet, all the clusterProfile objects MUST be placed in the same namespace. 
```

Comment:
The name of the namespace doesn't really matter if you have the label.

I think what's important with clusterset is guaranteeing that all clusters can be considered "having sameness" for a user, because that enables a lot of management simplicity. The requirement is that all clusters are part of the same cluster, not necessarily that they are all there.

use cases I can think of:
- regional management of clusterset; if one wanted localized management of clusters. Example: you have 3 clusters in europe and 3 clusters in us. They are part of the same clusterset but you want a regional hub-cluster. You would have 2 hub cluster that each manage 1 inventory of 3 clusters. The two regional clusters would have to be guaranteed by something global that they wont break sameness (maybe a global hub-cluster?).
- isolation of management: let's say we've given a inventory consumer the ability to manage N namespaces within some clusterProfiles, we could have an inventory of clusters thats a subset of the ClusterSet 
- migration: if we needed to migrate a clusterset, we will need to move one cluster at a time to the new representation of the clusterset (but without breaking the ClusterSet rules)
- timing of ON/OFF: clusters may come and go and join/leave the set as they are managed, so the completeness of the set may vary

Obviously a consumer of the API would be limited by the set of profiles they see and it may impact behaviors if the clusterset is not complete, but I think the main rule that we care about here is that the consumer can treat the clusters as being part of the same clusterset.